apps/api/docs/channels/sms-sns.md

```diff
@@ -0,0 +1,59 @@
+# AWS SNS
+
+Amazon Simple Notification Service (SNS) is a fully managed messaging service that supports application-to-application (A2A) and application-to-person (A2P) communication. Using AWS SNS, you can send SMS messages globally.
+
+Refer to the SNS documentation to dive deeper into the features AWS SNS offers and understand how it all works.
+
+## Values to Update in Database
+
+When using AWS SNS to send SMS via their API/Client, you need to provide certain variables that hold the SNS configuration details. Here are the values you need to update in table `notify_providers`:
+
+Create a new entry in table `notify_providers` and set the fields - `name`, `application_id`, `user_id`
+
+- Set field `channel_type` = 11 (for AWS SNS)
+- Set field `is_enabled` = 1 (to enable the newly created provider)
+
+Then set the following configurations in the `configuration` field
+
+| Key                   | Description                 |
+| --------------------- | --------------------------- |
+| AWS_ACCESS_KEY_ID     | Access key for AWS instance |
+| AWS_SECRET_ACCESS_KEY | Secret access key for AWS   |
+| AWS_REGION            | Region of the AWS instance  |
+
+```jsonc
+// Sample json to set in configuration field
+{
+    "AWS_ACCESS_KEY_ID":"Aws-Access-key",
+    "AWS_SECRET_ACCESS_KEY":"Aws-Secret-Key",
+    "AWS_REGION":"aws-region"
+}
+```
+
+### Sample Request Body
+
+Here's a sample request body:
```

Comment:
We should follow same request body format as other SMS providers. For example, we should be able to use similar format to this: https://github.com/OsmosysSoftware/osmo-x/blob/main/apps/api/docs/channels/sms-Twilio.md for this SNS SMS provider.
Update all relevant docs and code along with PR description.

Reply:
Updated